bluepysnap/circuit_validation.py

```diff
@@ -143,6 +145,8 @@ def _check_required_datasets(config):
         types_file = nodes_dict.get('node_types_file')
         if types_file is not None and not Path(types_file).is_file():
             errors.append(fatal('Invalid "node_types_file": {}'.format(types_file)))
+        if not nodes_dict.get('populations'):
```

Comment:
I thought it is optional, even in the new spec.

Reply:
Ah, yes. I think I reinforced this when I got the picture that this would be required in the BBP spec. I'll change it. We definitely need a PR to change the validation to only check the BBP spec.

Reply:
Okay, I discussed this with @jdcourcol  and populations indeed is required within BBP. I made a PR to update our documentation.

Reply:
ok